server/repository/src/db_diesel/name_tag.rs

```diff
@@ -73,4 +75,9 @@ impl NameTagFilter {
         self.store_id = Some(filter);
         self
     }
+
+    pub fn name_id(mut self, filter: EqualFilter<String>) -> Self {
+        self.name_id = Some(filter);
+        self
```

Comment:
Needed to add filtering by customer name to filter programSetting when a customer is selected